docs/PlaybackIDApi.md

```diff
@@ -0,0 +1,37 @@
+# \PlaybackIDApi
+
+All URIs are relative to *https://api.mux.com*
+
+Method | HTTP request | Description
+------------- | ------------- | -------------
+[**GetAssetOrLivestreamId**](PlaybackIDApi.md#GetAssetOrLivestreamId) | **Get** /video/v1/playback-ids/{PLAYBACK_ID} | Retrieve an Asset or Live Stream ID
+
+
+# **GetAssetOrLivestreamId**
+> GetAssetOrLiveStreamIdResponse GetAssetOrLivestreamId(ctx, pLAYBACKID)
```

Comment:
If this didn't show up in our markdown docs, I wouldn't care, but can someone explain why we got inverse capitalization going on here? And do we care?